kernel/include/memory/virtual/AnonVmDriver.h

```diff
@@ -4,14 +4,30 @@
 
 namespace Npk::Memory::Virtual
 {
+    enum class AnonFeature : uintptr_t
+    {
+        Demand = 1 << 0,
```

Comment:
why you left shift 1 of 0 positions? 
Is just for consistency with other declrations?